planning/scenario_planning/common/motion_velocity_optimizer/src/motion_velocity_optimizer.cpp

```diff
@@ -194,26 +221,30 @@ void MotionVelocityOptimizer::run()
 
   auto t_end = std::chrono::system_clock::now();
   double elapsed = std::chrono::duration_cast<std::chrono::nanoseconds>(t_end - t_start).count();
-  ROS_DEBUG("run: calculation time = %f [ms]", elapsed * 1.0e-6);
-  ROS_DEBUG("============================== run() end ==============================\n\n");
+  RCLCPP_DEBUG(get_logger(), "run: calculation time = %f [ms]", elapsed * 1.0e-6);
+  RCLCPP_DEBUG(
+    get_logger(), "============================== run() end ==============================\n\n");
 }
 
-autoware_planning_msgs::Trajectory MotionVelocityOptimizer::calcTrajectoryVelocity(
-  const autoware_planning_msgs::Trajectory & traj_input)
+autoware_planning_msgs::msg::Trajectory MotionVelocityOptimizer::calcTrajectoryVelocity(
+  const autoware_planning_msgs::msg::Trajectory & traj_input)
 {
   /* Find the nearest point to reference_traj */
   int input_closest = vpu::calcClosestWaypoint(
     traj_input, current_pose_ptr_->pose, planning_param_.delta_yaw_threshold);
   if (input_closest < 0) {
-    ROS_WARN("[velocity planner] cannot find closest waypoint for input trajectory");
+    RCLCPP_WARN(
+      get_logger(), "[velocity planner] cannot find closest waypoint for input trajectory");
```

Comment:
The new recommendation for more granularity here and below in logging would be 
```suggestion
      get_logger().get_child("velocity planner"), "cannot find closest waypoint for input trajectory");
```

Reply:
Thanks. Here is to add the node name, so just remove the [velocity_planner] string.